### ISSUE TITLE
Skip group name for pre-selected teams

### ISSUE THREAD

**Issue by stephenyeargin** [seatshare/seatshare-rails#224]
The last several users (and many more before that) are not creating groups. It may be because of the extra step.

If a user has selected a team from the teams page, it does not make a lot of sense to have them provide a group name as a barrier to creating their group. Give it a generic name like "<First Name>'s <Team> Group" and they can rename it later through the settings page if they would like.

The same is likely true for setting up a new group where you have to select which team (entity) you want to use when creating the group.

tl;dr Naming the group is an unnecessary hurdle to getting folks started with the application.
